keyboards/planck/keymaps/mjuma/keymap.c

```diff
@@ -0,0 +1,253 @@
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Ctrl | FN   | GUI  | Alt  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_GAMING] = LAYOUT_planck_grid(
+    KC_TAB,            KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,      KC_Y,      KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+    KC_GESC,           KC_A,    KC_S,    KC_D,    KC_F,    KC_G,      KC_H,      KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+    TD(TD_SHIFT_CAPS), KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,      KC_N,      KC_M,    KC_COMM, KC_DOT,  KC_SLSH, MT(MOD_RSFT, KC_ENT),
+    KC_LCTL, FN,       KC_LGUI, KC_LALT, LOWER,   KC_SPC,  KC_SPC,    RAISE,     KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+),
+
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      |RESET |DEBUG |      |      |      |      |DMREC1|DMREC2|      |      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |GAMING|QWERTY|      |      |      |      |DMPLY1|DMPLY2|Audoff|Aud on|      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |DMRSTP|      |Musoff|Mus on|MusMod|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |AG_TOG|      |             |      |      |Voice-|Voice+|      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_planck_grid(
+    _______, RESET,   DEBUG,     _______,   _______,  _______, _______, DM_REC1, DM_REC2,   _______,   _______,   _______,
+    _______, GAMING,  QWERTY,    _______,   _______,  _______, _______, DM_PLY1, DM_PLY2,   AU_OFF,    AU_ON,     _______,
+    _______, _______, _______,   _______,   _______,  _______, _______, DM_RSTP, _______,   MU_OFF,    MU_ON,     MU_MOD,
+    _______, _______, _______,   AG_TOGG,   _______,  _______, _______, _______, _______,   MUV_DE,    MUV_IN,    _______
+)
+
+};
+
+/* Layer Change Code
+ * Runs everytime the layer gets changed
+ */
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+/*
+ * Extend or override keyboard key functionality
+ * Return: true to have qmk process the key
+ */
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QWERTY:
+            if (record->event.pressed) {
+                set_single_persistent_default_layer(_QWERTY);
+            }
+            return false;
+            break;
+        case GAMING:
```

Comment:
To be honest, this may work better as a TG, rather than a default layer, as I suspect you don't want it persistent after power loss?

Reply:
Didn't about that feature! Swapped it out and it works just fine. Also lets me get rid of `process_record_user` since I don't need that any more.